scm/driver/harness/webhook.go

```diff
@@ -285,6 +327,28 @@ func convertAction(src string) (action scm.Action) {
 	}
 }
 
+func convertBranchAction(src string) (action scm.Action) {
+	switch src {
+	case "branch_created":
+		return scm.ActionCreate
+	case "branch_deleted":
+		return scm.ActionDelete
+	default:
+		return
```

Comment:
I'd lean towards returning an error if it's an unknown value? or are we doing the same for other things?

Reply:
returning unkown what other provider do